dashboard/test/ui/features/hoc2015.feature

```diff
@@ -0,0 +1,134 @@
+Feature: Hour of Code 2015 tutorial is completable
+
+  @no_ie @no_mobile
+  Scenario: Solving puzzle 1 in block mode
```

Comment:
`@no_ie @no_mobile` because I can't get droplet block-dragging to work on those browsers (see comment on the `I drag droplet block "{name}" to line {number}` step definition).